.travis.yml

```diff
@@ -17,9 +17,9 @@ before_install:
 
 matrix:
   include:
-  - rvm: 1.9.3
-  - rvm: 2.0
-  - rvm: 2.2
+  - rvm: 2.2.5
```

Comment:
At the moment, most workstations use `gem install inspec`, therefore we need to support more versions that chef. Therefore we should not change the old version behavior.